pydis_core/utils/paste_service.py

```diff
@@ -16,22 +16,38 @@
 _lexers_supported_by_pastebin: dict[str,list[str]] = {}
 
 
-class PasteResponse(TypedDict):
+class PasteResponse(BaseModel):
     """
     A successful response from the paste service.
 
-    args:
-        link: The URL to the saved paste.
-        removal: The URL to delete the saved paste.
+    Args:
+        link (:obj:`str`): The URL to the saved paste.
+        removal (:obj:`str`): The URL to delete the saved paste.
     """
 
     link: str
     removal: str
 
 
+class PasteFile(BaseModel):
+    """
+    A file to be pasted to the paste service.
+
+    Args:
+        content (:obj:`str`): The content of the file.
+        name (:obj:`str`): The file name of the file to upload. Defaults to ``""``
```

Comment:
```suggestion
        name (:obj:`str`): The name of the file to upload. Defaults to ``""``
```

Reply:
Thanks :D https://github.com/python-discord/bot-core/pull/188/commits/d087bec1160d46820f0dba29d14ab3a0f25d0efc will squash it down before merging